lib/projectParser.ts

```diff
@@ -205,4 +218,29 @@ class FileCache {
     this.linter = vhdlLinter;
   }
 
+}
+class FileCacheVerilog {
+  lintingTime: number;
```

Comment:
Is this used somewhere?

Reply:
The lintingTime. Yes in line 123. This seems to be some old artefact. (But unrelated to the PR.